drivers/display/ssd16xx.c

```diff
@@ -217,6 +208,32 @@ static int ssd16xx_update_display(const struct device *dev)
 	return ssd16xx_write_cmd(dev, SSD16XX_CMD_MASTER_ACTIVATION, NULL, 0);
 }
 
+static int ssd16xx_blanking_off(const struct device *dev)
+{
+	struct ssd16xx_data *data = dev->data;
+
+	LOG_DBG("");
+
+	if (data->blanking_on) {
+		ssd16xx_busy_wait(dev);
```

Comment:
why is busy wait needed?

Reply:
Good question. It seems like it isn't actually needed.